### ISSUE TITLE
Usage of `opts`

### ISSUE THREAD

**Issue by RobLoach** [jstransformers/jstransformer-opalcss#3]
It's used for three different things:
1. Options for `opal`
2. Extended options for PostCSS
3. The options for PostCSS's `.render()` call

We should split this up into specific definitions so that the user has control over them.

**Comment by tunnckoCore** [comment]
It's not transformer job. All of these things are options, so `opts` is the best place for them.

**Comment by RobLoach** [comment]
> All of these things are options, so opts is the best place for them.

Correct. And since all of these things are options, it's difficult to know exactly what the `options` are to be applied to. Yes, they're all options, but options for what?

If you want to pass an option in for the instantiation of `postcss-simple-vars`, then it will also be used for all the other places listed in the original post.
